src/components/character-relationships/spec.js

```diff
@@ -0,0 +1,183 @@
+import React from 'react';
+import test from 'tape';
+import { shallow } from 'enzyme';
+import spy from 'utils/spy';
+
+import ListItem from 'material-ui/lib/lists/list-item';
+import IconButton from 'material-ui/lib/icon-button';
+import Dialog from 'material-ui/lib/dialog';
+import FlatButton from 'material-ui/lib/flat-button';
+import TextField from 'material-ui/lib/text-field';
+
+import CharacterRelationships from './';
+
+test('CharacterRelationships', t => {
+    let expected, actual;
+
+    let relationships = [
+        {
+            _id: 'the first id',
+            avatar: 'https://en.gravatar.com/userimage/10460545/28033602a20a2b3e9cd4db965c3b43c4.jpeg',
+            name: 'Chris Harrington',
+            description: 'Father'
+        },
+        {
+            _id: 'the second id',
+            name: 'Bruce Wayne',
+            description: 'Batman'
+        }
+    ];
+
+    {
```

Comment:
What is the reason for all these scoped blocks? Neither a criticism nor a complaint - just curious about your thought process as most people don't make aggressive use of them yet.